pkg/helm/run.go

```diff
@@ -80,6 +83,17 @@ func Run(flags *hoflags.HelmOperatorFlags) error {
 	mgr, err := manager.New(cfg, manager.Options{
 		Namespace:          namespace,
 		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+		NewClient: func(cache cache.Cache, config *rest.Config, options crclient.Options) (crclient.Client, error) {
```

Comment:
Should the same be done for Go too?

Reply:
The unstructured client, which is used for Helm and Ansible but not Go, uses a typical client for reads by default. We want to ensure that a cache is used for reads to match Go's client setup.